### PR TITLE
Add camel-xchange wrapper

### PR DESCRIPTION
**Pull request by f2par0**
camel-xchange wrapper

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -2066,12 +2066,14 @@
...
     </feature>
-<!--    <feature name='camel-xchange' version='${project.version}' start-level='50'>-->
-<!--        <feature version='${project.version}'>camel-core</feature>-->
-<!--        <bundle dependency='true'>wrap:mvn:org.knowm.xchange/xchange-core/${xchange-version}</bundle>-->
-<!--        <bundle dependency='true'>wrap:mvn:com.github.mmazi/rescu/${rescu-version}</bundle>-->
-<!--        <bundle>mvn:org.apache.camel/camel-xchange/${project.version}</bundle>-->
-<!--    </feature>-->
+    <feature name='camel-xchange' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature>jackson</feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml:2071]: Doesn't seem to be used/needed

Comment (f2par0) [reply features/src/main/feature/camel-features.xml:2071]: it could be removed for compilation, but I doubt that it works at runtime since there are many dependencies of xchange-core that relates to jackson packages, but they are optional by default in the wrap protocol.

Comment (essobedo) [reply features/src/main/feature/camel-features.xml:2071]: Are-you sure that it is used at runtime? Because according to the command `mvn dependency:tree -Dscope=runtime` in the camel project, there is no such dependency:
```
[INFO] 
[INFO] --- dependency:3.6.1:tree (default-cli) @ camel-xchange ---
[WARNING] The artifact com.github.tomakehurst:wiremock:jar:3.0.1 has been relocated to org.wiremock:wiremock:jar:3.0.1
[INFO] org.apache.camel:camel-xchange:jar:4.4.1
[INFO] +- org.apache.camel:camel-support:jar:4.4.1:compile
[INFO] |  +- org.apache.camel:camel-api:jar:4.4.1:compile
[INFO] |  |  \- jakarta.xml.bind:jakarta.xml.bind-api:jar:4.0.1:compile
[INFO] |  |     \- jakarta.activation:jakarta.activation-api:jar:2.1.2:compile
[INFO] |  +- org.apache.camel:camel-management-api:jar:4.4.1:compile
[INFO] |  +- org.apache.camel:camel-util:jar:4.4.1:compile
[INFO] |  +- org.apache.camel:camel-util-json:jar:4.4.1:compile
[INFO] |  +- org.apache.camel:camel-xml-jaxp-util:jar:4.4.1:compile
[INFO] |  \- org.slf4j:slf4j-api:jar:2.0.12:compile
[INFO] +- org.knowm.xchange:xchange-core:jar:5.1.1:compile
[INFO] |  +- com.github.mmazi:rescu:jar:3.0:compile
[INFO] |  |  +- jakarta.ws.rs:jakarta.ws.rs-api:jar:3.1.0:compile
[INFO] |  |  \- oauth.signpost:signpost-core:jar:2.1.1:compile
[INFO] |  +- org.apache.commons:commons-lang3:jar:3.14.0:compile
[INFO] |  +- com.google.guava:guava:jar:33.0.0-jre:compile
[INFO] |  |  +- com.google.guava:failureaccess:jar:1.0.2:compile
[INFO] |  |  +- com.google.guava:listenablefuture:jar:9999.0-empty-to-avoid-conflict-with-guava:compile
[INFO] |  |  +- org.checkerframework:checker-qual:jar:3.41.0:compile
[INFO] |  |  +- com.google.errorprone:error_prone_annotations:jar:2.23.0:compile
[INFO] |  |  \- com.google.j2objc:j2objc-annotations:jar:2.8:compile
[INFO] |  +- io.github.resilience4j:resilience4j-ratelimiter:jar:1.7.0:compile
[INFO] |  |  +- io.vavr:vavr:jar:0.10.2:compile
[INFO] |  |  |  \- io.vavr:vavr-match:jar:0.10.2:compile
[INFO] |  |  \- io.github.resilience4j:resilience4j-core:jar:1.7.0:compile
[INFO] |  +- io.github.resilience4j:resilience4j-retry:jar:1.7.0:compile
[INFO] |  \- javax.annotation:javax.annotation-api:jar:1.3.2:compile
[INFO] \- org.knowm.xchange:xchange-binance:jar:5.1.1:compile

```

Comment (f2par0) [reply features/src/main/feature/camel-features.xml:2071]: Not sure they are used at runtime but probably since they are used in the dto packages, this is how I saw them :

```
karaf@root()> install wrap:mvn:org.knowm.xchange/xchange-core/5.1.1
Bundle ID: 107
karaf@root()> headers 107

Export-Package = 
        org.knowm.xchange;
                uses:="org.knowm.xchange.client,
                        org.knowm.xchange.dto.meta,
                        org.knowm.xchange.exceptions,
                        org.knowm.xchange.instrument,
                        org.knowm.xchange.service.account,
                        org.knowm.xchange.service.marketdata,
                        org.knowm.xchange.service.trade,
                        org.slf4j,
                        si.mazi.rescu",
        org.knowm.xchange.client;uses:="io.github.resilience4j.ratelimiter,io.github.resilience4j.retry,io.vavr.control,jakarta.ws.rs.core,org.knowm.xchange,si.mazi.rescu",
        org.knowm.xchange.currency;uses:="com.fasterxml.jackson.annotation,org.knowm.xchange.instrument",
        org.knowm.xchange.derivative;uses:="com.fasterxml.jackson.annotation,org.knowm.xchange.currency,org.knowm.xchange.instrument",
        org.knowm.xchange.dto;uses:="com.fasterxml.jackson.annotation,org.knowm.xchange.currency,org.knowm.xchange.dto.trade,org.knowm.xchange.instrument",
        org.knowm.xchange.dto.account;
                uses:="com.fasterxml.jackson.annotation,
                        com.fasterxml.jackson.databind.annotation,
                        org.knowm.xchange.currency,
                        org.knowm.xchange.instrument",
        org.knowm.xchange.dto.marketdata;
                uses:="com.fasterxml.jackson.annotation,
                        com.fasterxml.jackson.databind.annotation,
                        org.knowm.xchange.currency,
                        org.knowm.xchange.dto,
                        org.knowm.xchange.dto.trade,
                        org.knowm.xchange.instrument",
        org.knowm.xchange.dto.meta;
                uses:="com.fasterxml.jackson.annotation,
                        com.fasterxml.jackson.core,
                        com.fasterxml.jackson.databind,
                        com.fasterxml.jackson.databind.annotation,
                        org.knowm.xchange.currency,
                        org.knowm.xchange.dto.account,
                        org.knowm.xchange.instrument",
        org.knowm.xchange.dto.trade;
                uses:="com.fasterxml.jackson.annotation,
                        com.fasterxml.jackson.databind.annotation,
                        org.knowm.xchange.currency,
                        org.knowm.xchange.dto,
                        org.knowm.xchange.dto.marketdata,
                        org.knowm.xchange.instrument",
        org.knowm.xchange.exceptions;uses:="org.knowm.xchange.currency,org.knowm.xchange.instrument",
        org.knowm.xchange.instrument;uses:="com.fasterxml.jackson.databind.annotation,org.knowm.xchange.currency,org.knowm.xchange.utils.jackson",
        org.knowm.xchange.interceptor;uses:=si.mazi.rescu,
        org.knowm.xchange.service;
                uses:="io.github.resilience4j.ratelimiter,
                        io.github.resilience4j.retry,
                        javax.crypto,
                        org.knowm.xchange,
                        org.knowm.xchange.client,
                        org.knowm.xchange.dto,
                        org.knowm.xchange.dto.meta,
                        org.knowm.xchange.dto.trade,
                        si.mazi.rescu",
        org.knowm.xchange.service.account;
                uses:="org.knowm.xchange.currency,
                        org.knowm.xchange.dto.account,
                        org.knowm.xchange.instrument,
                        org.knowm.xchange.service,
                        org.knowm.xchange.service.trade.params",
        org.knowm.xchange.service.marketdata;
                uses:="org.knowm.xchange.currency,
                        org.knowm.xchange.dto.marketdata,
                        org.knowm.xchange.instrument,
                        org.knowm.xchange.service,
                        org.knowm.xchange.service.marketdata.params,
                        org.knowm.xchange.service.trade.params",
        org.knowm.xchange.service.marketdata.params;uses:="org.knowm.xchange.currency,org.knowm.xchange.instrument",
        org.knowm.xchange.service.trade;
                uses:="org.knowm.xchange.dto,
                        org.knowm.xchange.dto.account,
                        org.knowm.xchange.dto.trade,
                        org.knowm.xchange.service,
                        org.knowm.xchange.service.trade.params,
                        org.knowm.xchange.service.trade.params.orders",
        org.knowm.xchange.service.trade.params;
                uses:="javax.annotation,
                        org.knowm.xchange.currency,
                        org.knowm.xchange.dto,
                        org.knowm.xchange.dto.account,
                        org.knowm.xchange.instrument",
        org.knowm.xchange.service.trade.params.orders;
                uses:="org.knowm.xchange,
                        org.knowm.xchange.currency,
                        org.knowm.xchange.dto,
                        org.knowm.xchange.dto.trade,
                        org.knowm.xchange.instrument,
                        org.knowm.xchange.service.trade.params",
        org.knowm.xchange.utils;uses:="com.fasterxml.jackson.databind.exc,javax.net.ssl,org.knowm.xchange,org.knowm.xchange.dto,org.knowm.xchange.dto.meta",
        org.knowm.xchange.utils.jackson;uses:="com.fasterxml.jackson.core,com.fasterxml.jackson.databind,org.knowm.xchange.currency,org.knowm.xchange.instrument",
        org.knowm.xchange.utils.nonce;uses:=si.mazi.rescu,
        org.knowm.xchange.utils.timestamp
Import-Package = 
        com.fasterxml.jackson.annotation;resolution:=optional,
        com.fasterxml.jackson.core;resolution:=optional,
        com.fasterxml.jackson.databind;resolution:=optional,
        com.fasterxml.jackson.databind.annotation;resolution:=optional,
        com.fasterxml.jackson.databind.exc;resolution:=optional,

```

Comment (essobedo) [reply features/src/main/feature/camel-features.xml:2071]: We should keep the Camel project as the only reference otherwise it will be hard to follow. Keep in mind that Camel only uses a sub-part of the library not 100 % of it. 

But in this particular case, it looks like the Jackson dependencies are provided by `wiremock` with `test` scope which could explain why the tests pass so let's keep it like this

**Review by essobedo — APPROVED** [review]
LGTM
